k8s/mongodb-pvc.yml

```
@@ -1,12 +1,12 @@
-  name: mongo-pvc
+  name: mongodb-pvc
   namespace: chat-app
 spec:
   accessModes:
     - ReadWriteOnce
   resources:
     requests:
-      storage: 1Gi
-  storageClassName: standard
\ No newline at end of file
+      storage: 5Gi
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)